.gitignore

```diff
@@ -11,10 +11,10 @@ gradle-app.setting
 .gradletasknamecache
 
 # # Work around https://youtrack.jetbrains.com/issue/IDEA-116898
-gradle/wrapper/gradle-wrapper.properties
+# gradle/wrapper/gradle-wrapper.properties
```

Comment:
Yeah, IDEA wants to use gradle-all to get the src code for auto completion. not a big deal, just let it update the file and commit it.